src/bepasty/utils/upload.py

```diff
@@ -72,8 +82,8 @@ def filter_type(cls, ct, ct_hint, filename=None):
         if not ct and filename:
             ct, encoding = mimetypes.guess_type(filename)
         if not ct:
-            return ct_hint
-        return cls._type_re.sub('', ct)[:50]
+            return ct_hint, True
+        return cls._type_re.sub('', ct)[:50], False
```

Comment:
please document the meaning of the return value elements.

Reply:
added following comment

```python
        # Return value:
        # tuple[0] - content-type string
        # tuple[1] - whether tuple[0] is hint or not
        #            True:  content-type is just a hint
        #            False: content-type was specified by user
```